src/application.coffee

```diff
@@ -150,8 +150,12 @@ fetch = (app, setDeviceUpdateState = true) ->
 		device.updateState(status: 'Downloading', download_progress: 0)
 
 		Promise.try ->
-			JSON.parse(app.env)
-		.then (env) ->
+			# In dependent apps env is replaced with config
```

Comment:
It will because we switch it to the `fullEnv` before actually running the update. It's ugly.
So yeah, as soon as this is merged and I clear up the pipeline a bit I'll clean this mess up.